modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -208,19 +215,22 @@ Protobuf::
 | int64 | long
 | sint64 | long
 | sfixed32 | int
-| sfixed64 | int
+| sfixed64 | long
 | string | string
 | bytes | binary
 | map | map
 |===
 
+There are some cases where the Protobuf type does not map directly to an Iceberg type and Redpanda applies the following transformations:
+
 * Repeated values are translated into Iceberg `array` types.
 * Enums are translated into Iceberg `int` types based on the integer value of the enumerated type.
 * `uint32` and `fixed32` are translated into Iceberg `long` types as that is the existing semantic for unsigned 32-bit values in Iceberg.
 * `uint64` and `fixed64` values are translated into their Base-10 string representation.
 * The `timestamp` type in Protobuf is translated into `timestamp` in Iceberg.
```

Comment:
timestamp type in proto could be google.protobuf.Timestamp

Reply:
Changed `timestamp` to `google.protobuf.Timestamp`